packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -344,21 +375,21 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
   }
 
   private subscribeCallEvents(): void {
-    this.call?.on('remoteParticipantsUpdated', this.onRemoteParticipantsUpdated);
-    this.call?.on('isMutedChanged', this.isMyMutedChanged);
-    this.call?.on('isScreenSharingOnChanged', this.isScreenSharingOnChanged);
-    this.call?.on('idChanged', this.callIdChanged);
+    this.call?.on('remoteParticipantsUpdated', this.onRemoteParticipantsUpdated.bind(this));
+    this.call?.on('isMutedChanged', this.isMyMutedChanged.bind(this));
+    this.call?.on('isScreenSharingOnChanged', this.isScreenSharingOnChanged.bind(this));
+    this.call?.on('idChanged', this.callIdChanged.bind(this));
   }
 
   private unsubscribeCallEvents(): void {
     for (const subscriber of this.participantSubscribers.values()) {
       subscriber.unsubscribeAll();
     }
     this.participantSubscribers.clear();
-    this.call?.off('remoteParticipantsUpdated', this.onRemoteParticipantsUpdated);
-    this.call?.off('isMutedChanged', this.isMyMutedChanged);
-    this.call?.off('isScreenSharingOnChanged', this.isScreenSharingOnChanged);
-    this.call?.off('idChanged', this.callIdChanged);
+    this.call?.off('remoteParticipantsUpdated', this.onRemoteParticipantsUpdated.bind(this));
```

Comment:
Looks like these listeners are arrow functions already, do we wanna modify it to normal function(or keep it arrow without a binding?), might be good to check chat side as well

Reply:
Converted to normal functions.